src/app/Routes.js

```diff
@@ -18,6 +18,7 @@ import InitialPage from 'pages/InitialPage';
 
 import ForgotPasswordPage from 'pages/ForgotPasswordPage';
 import CreateCommunity from 'pages/CreateCommunity';
+import AddContributor from 'pages/AddContributor'; // Importando pagina de inclusão de usuário
```

Comment:
```suggestion
import AddContributor from 'pages/AddContributor';
```
Acho que não precisa do comentário nesse caso, só com o código já da pra entender tranquilo